README.md

```diff
@@ -0,0 +1,24 @@
+# Contains the models, world and launch files for the IMAV19 simulation
+
+## How to use
+
```

Comment:
provide build instructions first. not ROS installation, just building

Reply:
Done